android/src/main/java/vn/luongvo/kmm/survey/android/ui/screens/home/HomeScreen.kt

```diff
@@ -20,11 +18,22 @@ fun HomeScreen(
 @Composable
 private fun HomeScreenContent() {
     Box(
-        modifier = Modifier
-            .fillMaxSize()
-            // TODO demo background, integrate in https://github.com/luongvo/kmm-survey/issues/16
-            .background(Color(0xFF8E9398))
+        modifier = Modifier.fillMaxSize()
     ) {
+        HomeSurveys(
+            listOf(
```

Comment:
We might need `todo` here.

Reply:
Fixed in https://github.com/luongvo/kmm-survey/pull/61/commits/c9a6d2a223230e4cb658f59ea145231bf6df66f8